src/components/molecules/ZeroResults/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from "react";
+import { ZeroResultsProps } from "./interface";
+
+import * as S from "./styles";
+
+const ZeroResults: FC<ZeroResultsProps> = ({ setTag, setSearchTerm }) => {
+  const suggestions = ["Ciągi", "Jedzenie", "Zwierzęta"];
+
+  const handleOnClick = (el: string) => {
+    setTag(el);
+    setSearchTerm(el);
+  };
+
+  return (
+    <S.ZeroResultsWrapper>
+      <S.ZeroResultsImg aria-label="Dziewczyna patrząca przez lornetkę" />
+      <S.ZeroResultsHeader>Brak wyników</S.ZeroResultsHeader>
+      <S.ZeroResultsP>Spróbuj wyszukać:</S.ZeroResultsP>
```

Comment:
It's not a paragraph, a span or tiny heading ;)

Reply:
Right, I've changed it :)